### DISCUSSION TITLE
Organization project colabs

### DISCUSSION THREAD

**Discussion by FrameMuse** [community/community#10095] [Projects and Issues]
When creating a beta project in organization, if changing it to public and trying to add outside collaborators to it, it says `Unable to add some users`

**Comment by coryschwartz** [comment]
I am also having this problem. It would be great if it would give a little more detail about what the problem aside from just saying "no". I'm not sure if that could be done in a way that doesn't leak private information.

I'm not sure what to tell the contributors. Do they need to validate their email? Is there some security thing they haven't set up yet? some other kind of problem? Is it even possible to add external contributors at all? no idea.
